tests/runfiles/common.run

```diff
@@ -898,3 +898,7 @@ tests = ['log_spacemap_import_logs']
 pre =
 post =
 tags = ['functional', 'log_spacemap']
+
+[tests/functional/zpool_influxdb]
+tests = 'zpool_influxdb'
+tags = ['functional', 'metrics']
```

Comment:
It'd probably make sense to move these tests under `tests/functional/cli_user/` and run then as a normal unprivileged user.  (`user =`)

Reply:
good idea